mmdet3d/datasets/pipelines/transforms_3d.py

```diff
@@ -10,6 +10,46 @@
 from .data_augment_utils import noise_per_object_v3_
 
 
+@PIPELINES.register_module()
+class RandomDropPointsColor(object):
+    """Randomly set the color of points to all zeros.
+
```

Comment:
It seems that this is applied to all the points once the drop is executed. Do we need to support moderately drop some of the points? If it is unnecessary, we may add a description for that in detail, e.g., indicating this is a common practice in xxx.

Reply:
I only see this augmentation in PAConv's paper so I am not sure if it can be called a "common practice".

In my opinion, I think dropping some points' color while keeping others will harm the model's performance. The intra-object color consistency (same objects share similar color, e.g. office chairs are totally black) is an important clue for segmentors. If we only drop some of the points, then the color will be messy and I think it would be bad for the learning process. Dropping all points' color keeps this consistency (though the inter-object difference is destroyed).

I'm inclined to not support partially dropping. We can add this in the future if needed but not now. I will add a link to PAConv's implementation.